src/pages/index.tsx

```diff
@@ -3,6 +3,7 @@ export * from './SovereignTransfer';
 export * from './Disclaimer';
```

Comment:
This should be an `index.ts` considering you're not rendering any `JSX`.

Reply:
I am exporting ONLY TSX files, pages

Reply:
Cool. 👍